### ISSUE TITLE
Nodeserver connect/disconnect with no info

### ISSUE THREAD

**Issue by jimboca** [UniversalDevicesInc/polyglot-v2#76]
The nodeserver log just says over and over:
```
2019-11-07 10:02:06,578 [Interface ] [INFO ] Sent Connected message to Polyglot
2019-11-07 10:02:07,669 [Interface ] [INFO ] MQTT Connected with result code 0 (Success)
2019-11-07 10:02:07,673 [Interface ] [INFO ] MQTT Unexpected disconnection. Trying reconnect.
```
While polyglot log says:
```
11/7/2019, 10:03:24 [polyglot] info: MQTTS: Client Disconnected:
11/7/2019, 10:03:24 [polyglot] info: MQTTS: Client Connected:
```

Checked the machine and there were 2 copies of this nodeserver running.  Killed both of them, then tried to start from Polyglot UI and it would not start up.

**Comment by mkohanim** [comment]
Are you on 2.2.6?

With kind regards,

******************************
  Michel Kohanim
  CEO

  (p) 818.631.0333
  http://www.universal-devices.com<http://www.universal-devices.com/>
******************************

**Comment by jimboca** [comment]
I don't think 2.2.6 has been released for RPi?  Has it?

**Comment by mkohanim** [comment]
Ah, sorry.

With kind regards,

******************************************
  Michel Kohanim
  CEO

(p) 818.631.0333

  http://www.universal-devices.com
******************************************

________________________________

**Comment by Einstein42** [comment]
You have multiple versions of the same nodeserver running. It disconnects everytime the other one connects, then vice versa. kill all running versions of the nodeserver and then start it as normal.

**Comment by jimboca** [comment]
Right, I realized that but would be nice to know what was happening somehow.  Something like "Client Disconnect because another one connected" Others on the forum have reported similar issues and require complete restart to fix.   Also, not sure why I couldn't start it up again.
